pkg/operator/ceph/object/objectstore.go

```diff
@@ -1000,3 +1009,46 @@ func errorOrIsNotFound(err error, msg string, args ...string) error {
 	}
 	return errors.Wrapf(err, msg, args)
 }
+
+// ShouldUpdateZoneEndpointList checks whether zone endpoint list need to be updated or not
+func ShouldUpdateZoneEndpointList(zones []zoneType, desiredEndpointList []string, zoneName string) (bool, error) {
```

Comment:
@thotz any reason this function and `listsAreEqual` should be in the `objectstore.go` and not in a file in the `zone` dir? These functions feel zone specific not object-store specific.

Reply:
Got it. If you want to keep it the same go for it, same with if you want to move them those functions, it's your call.